test-suite/squidconf/bug4832.conf.instructions

```diff
@@ -0,0 +1 @@
+skip-unless-autoconf-defines HAVE_AUTH_MODULE_BASIC
```

Comment:
I am not aware of other dependencies like this in the _existing_ test-suite/squidconf/ configuration files. If you know of any, please share them, and I will add them to this PR (if it is not going to blow up its scope).